docs/content/contribute/how-to-contribute/_index.adoc

```diff
@@ -0,0 +1,29 @@
+
+---
+title: "How to contribute"
```

Comment:
This title section of "How to Contribute" doesn't seem to be in the correct format.  Final file view looks to be off.

Reply:
That's because GitHub can't render [frontmatter](https://gohugo.io/content-management/front-matter/). If you follow the README to [view the documentation locally](https://github.com/ODU-CGA-CubeSat/flight-software#view-the-documentation-locally), it should render correctly